ci/cloudbuild/builds/lib/integration.sh

```diff
@@ -196,3 +196,7 @@ function integration::ctest_with_emulators() {
     "./google/cloud/bigtable/ci/${EMULATOR_SCRIPT}" \
     "${cmake_out}" "${ctest_args[@]}" -L integration-test-emulator
 }
+
+# To run the integration tests we need to install the dependencies for the storage emulator
+export PATH="${HOME}/.local/bin:${PATH}"
```

Comment:
I'd move this up top to just after the `source module` lines to keep all the code that executes automatically in this file in one spot.

Reply:
Done.